packages/admin/test/e2e/cli.feature.ts

```diff
@@ -374,4 +374,206 @@ describe('CLI', () => {
       ).toThrow('Unknown arguments: mnemonic, providerUrl, not-existent-command');
     });
   });
+
+  describe('AirnodeRequesterRrpAuthorizer', () => {
+    const AdminRank = Object.freeze({
+      Unauthorized: 0,
+      Admin: 1,
+      SuperAdmin: 2,
+    });
+    const oisTitle = 'title';
+    const endpointName = 'endpoint';
+    const endpointId = ethers.utils.keccak256(
+      ethers.utils.defaultAbiCoder.encode(['string'], [`${oisTitle}_${endpointName}`])
+    );
+    const expirationTimestamp = 1947451793;
```

Comment:
Nit: You could parse the date from UTC string to make it more readable (and use `.getTime` to obtain the timestamp on line 417)